lines.css

```diff
@@ -138,4 +138,17 @@ body {
     color: black;
     font-size: 12px;
     font-family: "Montserrat", sans-serif;
+}
+
+.permament-tools {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.history-button {
+    height: 25px;
+    width: 25px;
```

Comment:
Do not you want to choose svg image for history actions? FontAwesome could help, or some other sites.